tauri-wallet/src-tauri/src/operations/delegate.rs

```diff
@@ -92,3 +95,33 @@ pub async fn undelegate_from_gateway(
     Err(e) => Err(format_err!(e)),
   }
 }
+
+#[tauri::command]
+pub async fn get_reverse_mix_delegations_paged(
+  state: tauri::State<'_, Arc<RwLock<State>>>,
+) -> Result<PagedReverseMixDelegationsResponse, String> {
+  let r_state = state.read().await;
+  let client = r_state.client()?;
+  match client
```

Comment:
Rather than using `match`, you could use `map_err` here, i.e. 
```rust
client
  .get_reverse_mix_delegations_paged(Addr::unchecked(client.address().as_ref()), None, None)
  .await
  .map_err(|err| format_err!(err))
```